website/www/site/content/en/contribute/_index.md

```diff
@@ -83,217 +76,136 @@ sudo apt-get install \
+- [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing this website
+  changes locally
+- [Go](https://golang.org) 1.12 or later installed for Go SDK development
+- Python 3.6, 3.7, and 3.8. Yes, you need all four versions installed.
 
-## Accounts and Permissions
+  pip, setuptools, virtualenv, and tox installed for Python development
 
-- [Beam issue tracker (JIRA)](https://issues.apache.org/jira/projects/BEAM/issues):
-  Anyone can access it and browse issues. Anyone can register an account and login
-  to create issues or add comments. Only contributors can be assigned issues. If
-  you want to be assigned issues, a PMC member can add you to the project contributor
-  group.  Email the [dev@ mailing list](/community/contact-us)
-  to ask to be added as a contributor in the Beam issue tracker, and include your ASF Jira username.
+- for large contributions, a signed [Individual Contributor License
+  Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
+  Software Foundation (ASF).
 
-- [Beam Wiki Space](https://cwiki.apache.org/confluence/display/BEAM/Apache+Beam):
-  Anyone has read access. If you wish to contribute changes, please create an account and request edit access on the
-  [dev@ mailing list](/community/contact-us) (include your Wiki account user ID).
+</div>
 
-- Pull requests can only be merged by a
-  [Beam committer](https://home.apache.org/phonebook.html?pmc=beam).
+</br>
 
-- [Voting on a release](https://www.apache.org/foundation/voting.html): Everyone can vote. Only
-  [Beam PMC](https://home.apache.org/phonebook.html?pmc=beam) members should mark their votes as binding.
+```
+sudo apt-get install \
+   openjdk-8-jdk \
+   python-setuptools \
+   python-pip \
+   virtualenv \
+   tox \
+   docker-ce
+```
 
-## Communication
+</br>
 
-All communication is expected to align with the [Code of Conduct](https://www.apache.org/foundation/policies/conduct).
+You also need to [install Go](https://golang.org/doc/install).
 
-Discussion about contributing code to Beam happens on the [dev@ mailing list](/community/contact-us/). Introduce yourself!
+</br>
 
-Questions can be asked on the [#beam channel of the ASF slack](/community/contact-us/). Introduce yourself!
+Once Go is installed, install goavro:
 
-## Additional resources
+</br>
 
-If you are contributing a `PTransform` to Beam, we have an extensive
-[PTransform Style Guide](/contribute/ptransform-style-guide).
+```
+$ export GOPATH=`pwd`/sdks/go/examples/.gogradle/project_gopath
+$ go get github.com/linkedin/goavro
+```
 
-If you are contributing a Runner to Beam, refer to the
-[Runner authoring guide](/contribute/runner-guide/)
+</br>
 
-Review [design documents](https://s.apache.org/beam-design-docs).
+Linux users should configure their machines for sudoless Docker.
 
-A great way to contribute is to join an existing effort. For the most
-intensive efforts, check out the [roadmap](/roadmap/).
+</div>
```

Comment:
It looks like this information is in two places, is that intentional?

Reply:
Yes, unfortunately mobile version is a bit different so I had discern and duplicate some informations

Reply:
It's not duplicated anymore